list.c

```diff
@@ -0,0 +1,83 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <stdlib.h>
+#include "list.h"
+
+typedef struct Element {
+    int value;
+    struct Element* next;
+} Element;
+
+int getElementValue(Element* element) {
+    return element->value;
+}
+
+Element* elementNext(Element* element) {
+    if (element == NULL || element->next == NULL) return NULL;
```

Comment:
```suggestion
    if (element == NULL || element->next == NULL) {
        return NULL;
    }
```
Вообще по стайлгайду надо так, но можно не переделывать